docs/sdk/src/guides/your_first_pallet/mod.rs

```diff
@@ -160,13 +161,12 @@
 #![doc = docify::embed!("./src/guides/your_first_pallet/mod.rs", first_test)]
 //!
 //! In the first test, we simply assert that there is no total issuance, and no balance associated
-//! with account `1`. Then, we mint some balance into `1`, and re-check.
+//! with ALICE's account. Then, we mint some balance into ALICE's, and re-check.
 //!
 //! As noted above, the `T::AccountId` is now `u64`. Moreover, `Runtime` is replacing `<T: Config>`.
 //! This is why for example you see `Balances::<Runtime>::get(..)`. Finally, notice that the
 //! dispatchables are simply functions that can be called on top of the `Pallet` struct.
-//!
-//! TODO: hard to explain exactly `RuntimeOrigin::signed(1)` at this point.
+// TODO: hard to explain exactly `RuntimeOrigin::signed(ALICE)` at this point.
```

Comment:
```suggestion
// TODO: hard to explain exactly `RuntimeOrigin::signed(Alice)` at this point.
```

Reply:
I would leave the proper case to not code only